js_modules/dagster-ui/packages/ui-core/src/instance/PoolTag.tsx

```diff
@@ -0,0 +1,39 @@
+import {Box, Icon, Tag, Tooltip} from '@dagster-io/ui-components';
+import {Link} from 'react-router-dom';
+
+import {CONCURRENCY_KEY_DETAILS_QUERY} from './InstanceConcurrency';
+import {useQuery} from '../apollo-client';
+import {
+  ConcurrencyKeyDetailsQuery,
+  ConcurrencyKeyDetailsQueryVariables,
+} from './types/InstanceConcurrency.types';
+
+export const PoolTag = ({pool}: {pool: string}) => {
+  const path = `/deployment/concurrency/${pool}`;
+  const {data} = useQuery<ConcurrencyKeyDetailsQuery, ConcurrencyKeyDetailsQueryVariables>(
+    CONCURRENCY_KEY_DETAILS_QUERY,
+    {
+      variables: {
+        concurrencyKey: pool,
+      },
+    },
+  );
+
+  return (
+    <Tag>
+      <Box flex={{gap: 4, alignItems: 'center'}}>
+        <Icon name="dynamic_feed" />
+        <Link to={path}>{pool}</Link>
+        {data?.instance.concurrencyLimit &&
+        data.instance.concurrencyLimit.configuredLimit === null ? (
+          <Tooltip
+            placement="top"
+            content="This pool currently does not have any slots configured."
+          >
+            <Icon name="check_warning" />
```

Comment:
- How about `warning_outline`? The contrast of the black icon is maybe a little jarring.
- Should the `Tag` be `intent="warning"` or something in the warning case?